### ISSUE TITLE
The definition of cutoff radius of density

### ISSUE THREAD

**Issue by yingxingcheng** [theochem/denspart#10]
https://github.com/theochem/denspart/blob/7ee05fb7be2c34c99dcea7fc128cf02f150c3eb1/src/denspart/mbis.py#L57-L61

Is the prefactor $\frac{\alpha_{Ai}}{2}$ missed in the definition of the cutoff radius of radial density? 

$$\rho_{Ai}^0(|\vec{r}|) = \int_{\Omega} f_{Ai}(\vec{r}) d\Omega=\frac{\alpha_{Ai} N_{Ai}}{2} e^{-\alpha_{Ai}|\vec{r}|}$$

with

$$f_{Ai}(\vec{r}) = \frac{N_{Ai} \alpha_{Ai}}{8\pi}e^{-\alpha_{Ai}|\vec{r}-\vec{R}_A|}$$

**Comment by tovrstra** [comment]
I also agree it would be better to include it (and document the method). Something along the following lines would be more fitting:

$$R_\text{cut}=\frac{1}{\alpha}\bigl(\ln(\rho_\text{cut}) + \ln 8\pi  - \ln N - 3\ln \alpha\bigr)$$

**Comment by yingxingcheng** [comment]
I think a minus sign is missing. $R_\text{cut}' = -R_\text{cut}$

**Comment by tovrstra** [comment]
P.S. Long overdue, but I fixed it in 0182d1a
